### DISCUSSION TITLE
Installation outside of the SteamApps common directory

### DISCUSSION THREAD

**Discussion by V00id** [MadDeCoDeR/Classic-RBDOOM-3-BFG#156] [Q&A] [answered]
Hello,

this is an install related question.

Is it possible to install the content of the file "https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/releases/download/1.3.2-BFA_patched/windows-32-bit.zip" outside of "C:\Program Files (x86)\Steam\SteamApps\common\DOOM 3 BFG Edition"?
I.e. "C:\MyGames\DoomBFA"

Reason: I dont want to mix up my SteamApps common folders with other files.

Maybe start DoomBFA.exe with a switch -doombfgedition_dir "C:\Program Files (x86)\Steam\SteamApps\common\DOOM 3 BFG Edition"

Thanks in advanced.

**Comment by MadDeCoDeR** [comment]
Yes, DOOM BFA is 100% portable. Heck, the flatpak version is quite literally use and plays with that concept

**Reply by V00id** [reply]
The program ends with error messages.

Here is my exact setup:
1. The win x64 package is extracted to folder "E:\Games\Doom3\DoomBFA"
2. The DOOM 3 BFG Edition, steam app id 208200, is installed in "E:\Games\Steam\SteamApps\common\DOOM 3 BFG Edition" (NOT default "C:\Program Files (x86)\Steam\SteamApps\common\DOOM 3 BFG Edition")

If I start "E:\Games\Doom3\DoomBFA\DoomBFA.exe" a popup window with messages and buttons comes up.

**It says "_default material not found" in the top line.**

Obviously it cannot find the steam installation folder for DOOM 3 BFG Edition.

Does the programm support divergent SteamApps common folders?

**Reply by MadDeCoDeR** [reply, marked as answer]
Wait, I though you will copy the game files to the new folder. 
Well, anyway. You can use the launch arguments +set fs_basePath (for the game's installation folder) and +set fs_appPath (for where the BFA is installed)

**Reply by V00id** [reply]
Ok, thanks, +set fs_basePath does the trick. I also used +set fs_appPath for the DoomBFA folder.

But there is one thing: In the seetings and other menus are entries with a "#" sign. Obviously it cannot find the language files in ".\base\strings".
The log file has written warnings, but nothing about language file.
See "System_menu_error.png"
![System_menu_error](https://github.com/user-attachments/assets/a89eea7b-3644-4af0-9da0-82e4491cf6f1)

**Reply by V00id** [reply]
For your info: If I copy "E:\Games\Doom3\DoomBFA\base\strings\english_bfa.lang" into "E:\Games\Steam\SteamApps\common\DOOM 3 BFG Edition\base\strings" it shows the menu entries properly.
